### ISSUE TITLE
itemType for figures?

### ISSUE THREAD

**Issue by mvolz** [citation-style-language/zotero-bits#79]
Crossref has an item-type called "component" which seems to mostly be for figures. I thought about type casting it as "artwork" but there's no "publicationTitle" field for "artwork" so I ended up doing "webpage". 

http://api.crossref.org/types/component/works retrieves a list of 'component' items so you can see what they look like

**Comment by bwiernik** [comment]
CSL already has an item type `figure`. I think it would be good for Zotero to create an _Image_ item type or similar, likely mapped to CSL `figure`, for storing non-artistic images (and related items).

**Comment by denismaier** [comment]
Can we close this, or remove it from this board? The feature seems to be there already in CSL as @bwiernik points out. So this should probably be discussed somewhere else.

**Comment by bdarcus** [comment]
So it's "rejected"?

**Comment by adam3smith** [comment]
We need to think a bit about how to handle Zotero-relevant tickets. Remember that this issue tracker originally was both for Zotero _and_ CSL (hence the name) and some Zotero-relevant tickets are still on it. As per bwiernik's comment, on the CSL end, this is rejected, yes; on the Zotero end this is still relevant.

**Comment by denismaier** [comment]
So, let's just take this from this discussion board, or move it to rejected.
And perhaps we can add an appropriate label?

(That way, the issue will still be open in the zotero-bits issue tracker.)

**Comment by bwiernik** [comment]
I think migrating issues out of this repo and into csl-evolution if they are deemed relevant to CSL might be good. Leaving just zotero issues here.

**Comment by bdarcus** [comment]
Yeah, some of these tickets are more about Zotero, and some are about both.

Maybe it's be better to have CSL-only tickets that link to the
zotero-bits where relevant, and exclude all of the latter from this
project board?
